.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+| Q | A
+| --- | ---
+| Bug fix? | [yes|no]
+| New feature? | [yes|no]
+| BC breaks? | [yes|no]
+| Deprecations? | [yes|no]
+| Tests pass? | [yes|no]
```

Comment:
As said: The tests pass field in the checklist is more dedicated to a yes/no for the new tests created and covered in the code of the PR. If no, it triggers a TODO to get the tests or rather the code right. See the documentation PR.

In other cases I updated it to shortly describe why travis fails on other stuff, but my additions pass ^^